frontend/src/components/ChatBox/ChatBox.tsx

```diff
@@ -232,11 +235,13 @@ function ChatBox({
 					</span>
 				</div>
 				<div className="control-buttons">
-					<ExportPDFLink
-						messages={messages}
-						emails={emails}
-						currentLevel={currentLevel}
-					/>
+					<Suspense fallback={<p>loading pdf library</p>}>
```

Comment:
Have you checked what this looks like? You should be able to simulate a long loading using a promise and a timeout.

You might be better off using a disabled, no-op button (possibly even a LoadingButton) as your fallback component.

Reply:
Good shout. The text before was invisible!

I've got it looking like this before the real component is loaded
![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/8f4b5db6-84b8-4c82-9855-908da47e353c)

Delaying the loading with this code 
```typescript
const ExportPDFLink = lazy(async () => {
	return await later(10000).then(
		() => import('@src/components/ExportChat/ExportPDFLink')
	);
});

function later(delay: number) {
	return new Promise(function (resolve) {
		setTimeout(resolve, delay);
	});
}
```

Reply:
while I was at it, I updated the hover colour of the export chat and reset level buttons, because it was practically identical to the default background colour.